docs/userGuide/syntax/lists.md

```diff
@@ -27,11 +27,11 @@
 * Item 2
   * Item 2.1 { icon="fas-file-code" }
   * Item 2.2
-* Item 3 { icon="fas-code-branch" }
+* Item 3 { text="3" }
   * Item 3.1 
-* Item 4 { icon="octicon-git-pull-request" }
+* Item 4 { text="Pull request" icon="octicon-git-pull-request" }
   * Item 4.1 { icon="mif-perm-media" }
-* Item 5 { icon="glyphicon-education" }
+* Item 5 { text="" icon="glyphicon-education" }
```

Comment:
I was thinking that it would be good to show how to disable the text because of the cascading effect - if they don't want the rest of the bullets to have text, they would have to do something like this.

Reply:
(can we put a code comment...? hmm)